jupyter_scheduler/tests/test_handlers.py

```diff
@@ -1,13 +1,262 @@
 import json
+from contextlib import nullcontext
+from unittest.mock import patch
 
+import pytest
+from tornado.httpclient import HTTPClientError
 
-async def test_get_example(jp_fetch):
-    # When
-    response = await jp_fetch("jupyter-scheduler", "get_example")
+from jupyter_scheduler.handlers import compute_sort_model
+from jupyter_scheduler.models import (
+    CountJobsQuery,
+    DescribeJob,
+    ListJobsQuery,
+    ListJobsResponse,
+    SortDirection,
+    SortField,
+    Status,
+)
+from jupyter_scheduler.tests.utils import expected_http_error
+
+
+@pytest.mark.parametrize(
+    "job_id,payload",
+    [
+        (
+            "542e0fac-1274-4a78-8340-a850bdb559c8",
+            {
+                "input_uri": "notebook_a.ipynb",
+                "output_prefix": "outputs",
+                "idempotency_token": "",
+                "runtime_environment_name": "",
+            },
+        ),
+        (
+            "4c6cd4e0-49ce-4b58-843d-2fa02f7468b1",
+            {
+                "input_uri": "notebook_b.ipynb",
+                "output_prefix": "scheduled_outputs",
+                "idempotency_token": "",
+                "runtime_environment_name": "",
+                "name": "Scheduled Notebook B",
+                "job_definition_id": "7790f93c-4c2c-41b2-9085-daa93915d81c",
+                "parameters": {"a": 1, "b": 2, "foo": "bar", "test": True},
+            },
+        ),
+    ],
+)
+async def test_post_scheduled_jobs(jp_fetch, job_id, payload):
+    with patch("jupyter_scheduler.orm.uuid4") as mock_uuid:
+        mock_uuid.return_value = job_id
+        response = await jp_fetch("scheduler", "jobs", method="POST", body=json.dumps(payload))
+
+        assert response.code == 200
+        body = json.loads(response.body)
+        assert body["job_id"]
```

Comment:
Shouldn't this be:

```
assert body["job_id"] == job_id
```

Reply:
Good catch. Thanks for pointing this out.